pkg/boot/kexec/memory_linux.go

```diff
@@ -667,3 +667,52 @@ func (m *MemoryMap) Insert(r TypedRange) {
 	newMap.sort()
 	*m = newMap
 }
+
+// Memory type (uint32) in UefiPayload
+const (
+	PayloadTypeRAM      = 1
```

Comment:
Can you do a bit of work to change the above `type RangeType string` to a `type RangeType int`, so the underlying type of the enum is an int. Then add a `String()` for converting to a string. This would prevent us from having a number of duplicate types and maps.

Reply:
The issue is that this is used in other place. My plan is later we do a refactor PR on this file.

Reply:
agree on refactor in another PR